### ISSUE TITLE
Request for accessibility features

### ISSUE THREAD

**Issue by psubhashish** [Common-Voice/sentence-collector#234]
The thumbs up and thumbs down emojis are not very accessibility-friendly. When checked for thumbs up the green background and the yellow-orange emoji look pretty confusing. Also the thumbs up and thumbs down icon look very similar and reduced the speed of reviewing especially when one can otherwise do it faster.

I'd suggest for high contract (preferably black and white) icons and background like below. In a philosophical level, Common Voice is going to address an accessibility issue and having inaccessible UI is ironical.
![Accessible_UI](https://user-images.githubusercontent.com/1258090/60400187-54b01d00-9b8e-11e9-871d-28a9bf2fe773.png)
* Colors used in the second option: Greed (#096D53), Red (#9B0404)
* Icons: Designed by Photoroyalty / Freepik
